website/source/docs/operating-a-job/update-strategies/blue-green-and-canary-deployments.html.md

```diff
@@ -17,136 +16,438 @@ organizations prefer to put a "canary" build into production or utilize a
-desired set. Here is a sample job file which models the transition from version
-1.3 to version 1.4 using a blue/green deployment.
+To start, we examine our job which is running in production:
 
 ```hcl
 job "docs" {
-  datacenters = ["dc1"]
+  ...
 
-  group "api-green" {
-    count = 10
+  group "api" {
+    count = 5
 
     task "api-server" {
       driver = "docker"
 
-      config {
-        image = "api-server:1.3"
+      update {
+        max_parallel     = 1
+        canary           = 5
+        min_healthy_time = "30s"
+        healthy_deadline = "10m"
+        auto_revert      = true
       }
-    }
-  }
-
-  group "api-blue" {
-    count = 0
-
-    task "api-server" {
-      driver = "docker"
 
       config {
-        image = "api-server:1.4"
+        image = "api-server:1.3"
       }
     }
   }
 }
 ```
 
-It is clear that the active group is "api-green" since it has a non-zero count.
-To transition to v1.4 (api-blue), we increase the count of api-blue to match
-that of api-green.
+We see that it has an `update` stanza that has the `canary` equal to the desired
+count. This is what allows us to easily model blue/green deployments. When we
+change the job to run the "api-server:1.4" image, Nomad will create 5 new
+allocations without touching the original "api-server:1.3" allocations. Below we
+can see how this works by changing the image to run the new version:
 
 ```diff
 @@ -2,6 +2,8 @@ job "docs" {
- group "api-blue" {
--  count = 0
-+  count = 10
-
-   task "api-server" {
-     driver = "docker"
+  group "api" {
+    task "api-server" {
+      config {
+-       image = "api-server:1.3"
++       image = "api-server:1.4"
 ```
 
 Next we plan and run these changes:
 
 ```shell
 $ nomad plan docs.nomad
++/- Job: "docs"
++/- Task Group: "api" (5 canary, 5 ignore)
+  +/- Task: "api-server" (forces create/destroy update)
+    +/- Config {
+      +/- image: "api-server:1.3" => "api-server:1.4"
+        }
+
+Scheduler dry-run:
+- All tasks successfully allocated.
+
+Job Modify Index: 7
+To submit the job with version verification run:
+
+nomad run -check-index 7 example.nomad
+
+When running the job with the check-index flag, the job will only be run if the
+server side version matches the job modify index returned. If the index has
+changed, another user has modified the job and the plan's results are
+potentially invalid.
+
+$ nomad run docs.nomad
+...
 ```
 
-Assuming the plan output looks okay, we are ready to run these changes.
+We can see from the plan output that Nomad is going to create 5 canaries that
+are running the "api-server:1.4" image and ignore all the allocations running
+the older image. Now if we examine the status of the job we can see that both
+the blue ("api-server:1.3") and green ("api-server:1.4") set are running.
 
 ```shell
-$ nomad run docs.nomad
+$ nomad status docs
+ID            = docs
+Name          = docs
+Submit Date   = 07/26/17 19:57:47 UTC
```

Comment:
This might sound silly, but this is a reflection of the last time the page was updated. What are your thoughts on making this something like 01/01/01 00:00:00 UTC or some fixed time?